package-lock.json

```diff
@@ -0,0 +1,16693 @@
+{
+  "name": "cra-template-typescript-redux",
```

Comment:
If you're using npm it's totally fine to commit `package-lock.json`. I'm using `yarn` which has its own lock file `yarn.lock` and it's not recommended to have both of them, hence mixed up `npm` and `yarn`.